src/test/groovy/com/heroku/sdk/gradle/HerokuPluginTest.groovy

```diff
@@ -16,11 +16,10 @@ class HerokuPluginTest extends Specification {
     File projectDir
     File buildFile
     String appName
-    List<File> pluginClasspath
 
     GradleRunner with(String... tasks) {
         GradleRunner.create()
-                .withPluginClasspath(pluginClasspath)
+                .withPluginClasspath()
```

Comment:
The old `pluginClasspath` setup is no longer necessary when using `java-gradle-plugin`